lib/dal.js

```diff
@@ -12,26 +29,32 @@ var customError = require('./customError.js');
 var getSharedExtension = function(confFile, name) {
   var defer = Q.defer();
   var sharedExtension = {};
-
   sharedExtension.found = false;
-  return fetchJSON(confFile).then(function(data) {
-    data.sharedExtensions.forEach(function(extension, i) {
+  fetchJSON(confFile).then(function(data) {
+    data.sharedExtensions.some(function(extension, i) {
```

Comment:
I'm not sure where you are getting some(). It's not a built in JavaScript function and it looks like you are calling it on a built in array. It's part of lodash for example but you're not using it the way it's intended there.